Frontend/ui-customer/src/Services/Backend.purs

```diff
@@ -1446,8 +1447,79 @@ mkRentalSearchReq slat slong dlat dlong srcAdd desAdd startTime estimatedRentalD
                     "fareProductType" : "RENTAL"
                    }
 
-------------------------------------------------------------------------- Edit Location API -----------------------------------------------------------------------------
+makeAmbulanceSearchReq :: Number -> Number -> Number -> Number -> Address -> Address -> String -> Boolean -> Boolean -> String -> Boolean -> SearchReq
+makeAmbulanceSearchReq slat slong dlat dlong srcAdd desAdd startTime sourceManuallyMoved destManuallyMoved sessionToken isSpecialLocation = 
+    let appConfig = CP.getAppConfig CP.appConfig
+    in  SearchReq 
+        { "contents" : AmbulanceSearchRequest 
```

Comment:
Can we unify this with `makeRideSearchReq` as this is totally similar just a check on `FareproductType`